src/store/reducers/userReducer.ts

```diff
@@ -85,6 +87,7 @@ const initialState = {
   idData: {
     id: "",
   },
+  delete: { id: "" },
```

Comment:
id is not string

Reply:
I have changed the initial state to undefined